UI/styles/helper.css

```diff
@@ -0,0 +1,210 @@
+*,*::before,*::after{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+header{
+    display: flex;
+    flex-direction: row;
+    -ms-flex-direction: row;
+    position: relative;
+    height: 80px;
+    left: 0px;
+    top: 0px;
+    height: 80px;
+    z-index: 1;
+    justify-content: space-between;
+}
+#current-nav > a{
+    left: 0%;
+    right: 0%;
+    top: 0%;
+    bottom: 0%;
+    text-align: center;
+    background: linear-gradient(180deg, #DF502A 0%, rgba(35, 112, 172, 0.68) 100%);
+    box-shadow: 2px 2px 5px 1px #FEFCFC;
+    border-radius: 5px;
+}
+.sticky{
+    height: 80px;
+    width: 100%;
+    position: fixed;
+}
+.navbar ul{
+    display: flex;
+    flex-direction: row;
+    -ms-flex-direction: row;
+}
+.navbar ul li{
+    display:flex ;
+    list-style: none;
+    margin: 10px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+.navbar a{
+    width: 90px;
+    height: 32px;
+    top: 33px;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 28px;
+    color: #F5F5F5;
+    text-decoration: none;
+}
+.navbar a:hover{
+    cursor: pointer;
+    color: rgb(196, 190, 190);
+}
+#logo > img{
+    width: 70px;
+    height: 70px;
+    margin-left: 10px;
+    margin-top: 5px;
+    border-radius: 100%;
+}
+.toggle-switch{
+    margin-top: 10px;
+    float: right;
```

Comment:
With display flex, you'll not have to use floats, perhaps justify. Please explore these options also

Reply:
Definitely  I found that to position right with display flex, I should have used `margin-left: auto`.